index.src.html

```diff
@@ -70,6 +70,9 @@
       text: Navigator
 spec: HTML; urlPrefix: https://html.spec.whatwg.org/multipage/
   type: dfn
+    text: same-origin; url: concept-origin-effective-domain
+    text: origin; url: concept-origin
+    text: ASCII serialization; url: ascii-serialisation-of-an-origin
```

Comment:
I'm afraid I'm holding it wrong:
&lt;a lt="ASCII serialization of an origin"&gt;ASCII serialization&lt;/a&gt;
results in no errors, but the text also is not linked to anything.